selfdrive/manager/process_config.py

```diff
@@ -87,6 +87,7 @@ def only_offroad(started, params, CP: car.CarParams) -> bool:
   # debug procs
   NativeProcess("bridge", "cereal/messaging", ["./bridge"], notcar),
```

Comment:
@fredyshox Can we remove this then?

Reply:
No, this is bridge for body -> pc communication. The one you need to setup manually is for pc -> body side.